src/renderer/renderer.cpp

```
@@ -1,14 +1,35 @@
+  case VK_DEBUG_UTILS_MESSAGE_SEVERITY_ERROR_BIT_EXT:
+    MR_ERROR("{}\n", CallbackData->pMessage);
+    break;
+  case VK_DEBUG_UTILS_MESSAGE_SEVERITY_WARNING_BIT_EXT:
+    MR_WARNING("{}\n", CallbackData->pMessage);
+    break;
+  default:
+    MR_INFO("{}\n", CallbackData->pMessage);
+    break;
+  }
+  return false;
+}
```

Cppcheck (2.10-2)

[performance] Variable '_vbufs' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)